app.py

```diff
@@ -21,14 +21,14 @@ def predict():
     #predict = prediction[:, 1]

```

Comment:
GPT summary of [1fcf8a](https://github.com/SampatSuthar/ml-demo/blob/f3fc2c559b5eac826cd89d51e16fef43ebaff446/app.py#1fcf8aed8558b2618a97636c2f6d6c3587c2a6ee) - [036648](https://github.com/SampatSuthar/ml-demo/blob/2740f769fa9a434b94ae0ad30da100a387043dc3/app.py#0366480d84d3b97095bd42f20f2d7a550eadb09b):
* Removed import of openai and the corresponding API key
* Changed model from "text-davinci-003" to "davinci-003"
* Changed variable from "generated_text" to "generated"
* Updated print statement to print the new variable